benchmark/pom.xml

```diff
@@ -39,7 +39,7 @@
         <dependency>
             <groupId>io.prometheus</groupId>
             <artifactId>client</artifactId>
-            <version>0.0.11-SNAPSHOT</version>
+            <version>0.0.10</version>
```

Comment:
Ok, another question because I have no clue what's going on here. Why is there still some dependency section here at all for the "client", which is being deleted in this PR?